utils/constants.tsx

```diff
@@ -1027,8 +1027,6 @@ export const DefaultRolePermissions = {
         Permissions.MANAGE_PRIVATE_CHANNEL_MEMBERS,
         Permissions.DELETE_POST,
         Permissions.EDIT_POST,
-        Permissions.MANAGE_CREATE_EMOJIS,
-        Permissions.MANAGE_DELETE_EMOJIS,
```

Comment:
Best as I can tell, `MANAGE_CREATE_EMOJIS` and `MANAGE_DELETE_EMOJIS` have been undefined from the time they were introduced in this 2019 commit: https://github.com/mattermost/mattermost-webapp/commit/cc2d3b6f34aafdb376a78675ddd6229f652c8629